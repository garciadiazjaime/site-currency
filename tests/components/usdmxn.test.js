import React from 'react'
import { shallow } from 'enzyme'

import Usdmxn from '../../components/usdmxn'

describe('Usdmxn', () => {
  describe('main', () => {
    it('matches snapshot', () => {
      const wrapper = shallow(<Usdmxn />)
  
      expect(wrapper).toMatchSnapshot()
    })

    it('renders props when passed', () => {
      const entity = 'entity'
      const buy = 'buy'
      const sale = 'sale'

      const wrapper = shallow(<Usdmxn entity={entity} buy={buy} sale={sale} />)
  
      expect(wrapper).toMatchSnapshot()
    })
  })
})
