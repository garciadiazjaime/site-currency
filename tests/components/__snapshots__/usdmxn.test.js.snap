// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Usdmxn main matches snapshot 1`] = `
<div>
  : 
   / 
</div>
`;

exports[`Usdmxn main renders props when passed 1`] = `
<div>
  entity
  : 
  buy
   / 
  sale
</div>
`;
